dubbo-common/src/main/java/org/apache/dubbo/common/bytecode/ClassGenerator.java

```diff
@@ -46,288 +47,345 @@
  * ClassGenerator
  */
 public final class ClassGenerator {
-    private static final AtomicLong CLASS_NAME_COUNTER = new AtomicLong(0);
-    private static final String SIMPLE_NAME_TAG = "<init>";
-    private static final Map<ClassLoader, ClassPool> POOL_MAP = new ConcurrentHashMap<ClassLoader, ClassPool>(); //ClassLoader - ClassPool
-    private ClassPool mPool;
-    private CtClass mCtc;
-    private String mClassName, mSuperClass;
-    private Set<String> mInterfaces;
-    private List<String> mFields, mConstructors, mMethods;
-    private Map<String, Method> mCopyMethods; // <method desc,method instance>
-    private Map<String, Constructor<?>> mCopyConstructors; // <constructor desc,constructor instance>
-    private boolean mDefaultConstructor = false;
-
-    private ClassGenerator() {
-    }
-
-    private ClassGenerator(ClassPool pool) {
-        mPool = pool;
-    }
-
-    public static ClassGenerator newInstance() {
-        return new ClassGenerator(getClassPool(Thread.currentThread().getContextClassLoader()));
-    }
-
-    public static ClassGenerator newInstance(ClassLoader loader) {
-        return new ClassGenerator(getClassPool(loader));
-    }
-
-    public static boolean isDynamicClass(Class<?> cl) {
-        return ClassGenerator.DC.class.isAssignableFrom(cl);
-    }
-
-    public static ClassPool getClassPool(ClassLoader loader) {
-        if (loader == null)
-            return ClassPool.getDefault();
 
-        ClassPool pool = POOL_MAP.get(loader);
-        if (pool == null) {
-            pool = new ClassPool(true);
-            pool.appendClassPath(new LoaderClassPath(loader));
-            POOL_MAP.put(loader, pool);
-        }
-        return pool;
-    }
-
-    private static String modifier(int mod) {
-        StringBuilder modifier = new StringBuilder();
-        if (Modifier.isPublic(mod)) modifier.append("public");
-        if (Modifier.isProtected(mod)) modifier.append("protected");
-        if (Modifier.isPrivate(mod)) modifier.append("private");
-
-        if (Modifier.isStatic(mod)) modifier.append(" static");
-        if (Modifier.isVolatile(mod)) modifier.append(" volatile");
+  private static final AtomicLong CLASS_NAME_COUNTER = new AtomicLong(0);
+  private static final String SIMPLE_NAME_TAG = "<init>";

```

Comment:
This is not the code format we need.

Reply:
use 4 blanks instead.

Reply:
![image](https://user-images.githubusercontent.com/18657682/46007014-853cc500-c0eb-11e8-8169-05d34dce455b.png)
but he told me 2 blanks

Reply:
what is the plugin you used? It's all ok on my mac.

Reply:
I have git hook，it's conflict